resources/js/index.js

```diff
@@ -47,3 +46,29 @@ function toggle_complete(element) {
   });
   sessionStorage.setItem("list", list_task);
 }
+
+function todo_list() {
+  let element = save_task();
+  let string_to_date = moment(element.due_date).format();
+  const formatted_due_date = moment(string_to_date).format("L");
+  const formatted_creation_date = moment(element.creation_date).format("L");
+  const listContainer = document.getElementById("list1");
+
```

Comment:
Could you explain what means this type of format, please?

Reply:
do you mean the L format? If that's the case, then, according to the Moment library, L means showing the date like this: "05/20/2019"